tests/features/rhdm/kieserver/rhdm-kieserver.feature

```diff
@@ -17,7 +17,7 @@ Feature: RHDM KIE Server configuration tests
   Scenario: deploys the hellorules example, then checks if it's deployed.
     Given s2i build https://github.com/jboss-container-images/rhdm-7-openshift-image from quickstarts/hello-rules/hellorules using rhdm71-dev
       | variable                        | value                                                                                        |
-      | KIE_SERVER_CONTAINER_DEPLOYMENT | rhdm-kieserver-hellorules=org.openshift.quickstarts:rhdm-kieserver-hellorules:1.4.0-SNAPSHOT |
+      | KIE_SERVER_CONTAINER_DEPLOYMENT | rhdm-kieserver-hellorules=org.openshift.quickstarts:rhdm-kieserver-hellorules:1.7.0-SNAPSHOT |
```

Comment:
why increase it?

Reply:
Because is changed to build correctly by maven inside the s2i in this PR https://github.com/jboss-container-images/rhdm-7-openshift-image/pull/366